contracts/lido_terra_hub/src/contract.rs

```diff
@@ -192,6 +171,80 @@ pub fn execute(deps: DepsMut, env: Env, info: MessageInfo, msg: ExecuteMsg) -> S
     }
 }
 
+pub fn execute_claim_airdrops(
+    deps: DepsMut,
+    _env: Env,
+    info: MessageInfo,
+    token: String,
+    stage: u8,
+    amount: Uint128,
+    proof: Vec<String>,
+) -> StdResult<Response> {
+    let params: Parameters = PARAMETERS.load(deps.storage)?;
+    if params.paused.unwrap_or(false) {
+        return Err(StdError::generic_err("the contract is temporarily paused"));
+    }
+
+    let config = CONFIG.load(deps.storage)?;
+    let owner = deps.api.addr_humanize(&config.creator)?;
+
+    if info.sender != owner {
+        return Err(StdError::generic_err("unauthorized"));
+    }
+
+    let withdrawal_account = deps.api.addr_humanize(
+        &config
+            .airdrop_withdrawal_account
+            .ok_or_else(|| StdError::generic_err("no withdrawal account configured"))?,
```

Comment:
This lambda is not needed here. I suggest replacing call to `ok_or_else()` with `.ok_or(StdError::generic_err("no withdrawal account configured")`

Reply:
Clippy check fails when used with ok_or, explanation - https://rust-lang.github.io/rust-clippy/master/index.html#or_fun_call